aspnetcore/blazor/security/passkeys.md

```diff
@@ -0,0 +1,663 @@
+---
+title: Enable Web Authentication API (WebAuthn) passkeys in an ASP.NET Core Blazor Web App
```

Comment:
Unlike the other document with detailed code examples, this document is a little bit more agnostic to the UI rendering technology, so I wonder if the title should refer to "ASP.NET Core Identity" instead of "ASP.NET Core Blazor Web App".

There are sections that include snippets from the Blazor Web App template, but the referenced code is not Blazor-specific and there isn't any UI logic.

Reply:
This goes to the ❓s that I asked @danroth27 on content layout. See my OP at the top ☝️.

Reply:
I think we want core security & identity concepts (auth, account confirmation, 2FA, passkeys, etc) to be covered in the top-level security section and Blazor specific security concepts to be covered in the Blazor security section. These security and identity concepts should be explained independently of any UI stack. In some cases, core security & identity features require some UI elements, like QR codes, forms, etc. For demonstration purposes, sample implementations of these UI elements should be provided in Blazor, but Blazor is not core to these features. Users could instead decide to implement the UI elements in MVC or Razor Pages. So yeah, I think we should break the pattern for content that isn't really Blazor specific and move that content into the top-level security node.

Reply:
I'll aim to organize along those lines on the next commit.